main/accounts/index.ts

```diff
@@ -105,7 +106,8 @@ export class Accounts extends EventEmitter {
       log.info(`Account ${address} not found, creating account`)
 
       const created = 'new:' + Date.now()
-
+      const accountMetaId = uuidv5(address, accountNS)
```

Comment:
I think the hashing should probably be done in the store action to prevent us from ever accidentally storing the plain-text value

Reply:
This is retrieving a value